clients/drcachesim/tools/invariant_checker.cpp

```diff
@@ -661,3 +629,61 @@ invariant_checker_t::print_results()
     std::cerr << "Trace invariant checks passed\n";
     return true;
 }
+
+std::string
```

Comment:
Preserve the comment  "Invariant: non-explicit control flow (i.e., kernel-mediated) is indicated by markers." somewhere: right now there is no mention of markers (nor of "non-explicit") except suddenly in the error message.